flytekit/bin/entrypoint.py

```diff
@@ -185,6 +180,17 @@ def _dispatch_execute(
         exit(1)
 
 
+def get_traceback_str(e: Exception) -> str:
+    tb = e.__cause__.__traceback__ if e.__cause__ else e.__traceback__
```

Comment:
should we make this recursive?

Are you happy with the way these look?  should we try to format them better?  This is only used in actual runs right?